txpool/txpool.go

```diff
@@ -203,6 +203,27 @@ func (t *TxPool) addImpl(ctx string, tx *types.Transaction) error {
 	return nil
 }
 
+// GetTxs gets both queued and pending transactions
+func (t *TxPool) GetTxs() (map[types.Address]map[uint64]*types.Transaction, map[types.Address]map[uint64]*types.Transaction) {
+	queuedTxs := make(map[types.Address]map[uint64]*types.Transaction)
+	queue := t.queue
+	for addr, queuedTxn := range queue {
+		for _, tx := range queuedTxn.txs {
+			queuedTxs[addr] = make(map[uint64]*types.Transaction)
```

Comment:
You might want to create new map outside second loop (between line 210 and 211). In current implementation, result shows only one transaction per an account. (According to doc, you can see transactions by multiple nonce if exist)

Reply:
Thanks @Kourin1996 , for both these comments, I created a new map only if one at the particular address doesn't exist. I found out that the scenario exists where a particular address might not have any txs, so this approach ensures there's always at least 1 tx per address.